app/controllers/application_controller.rb

```diff
@@ -260,9 +260,9 @@ def forward_to_app_config_domain
     # Let's only redirect get requests for this purpose.
     return unless request.get? &&
       # If the request equals the original set domain, e.g. forem-x.forem.cloud.
-      request.host == ENV["APP_DOMAIN"] &&
+      request.host == ENV.fetch("APP_DOMAIN", nil) &&
```

Comment:
There's a bunch of stuff that's popped up as autofixes from when I merged `main` into my branch 🤷‍♀️  I'm not sure how they didn't get caught in their original PRs, but whenever I try to change them back they just autofix themselves again 😂